<?xml version="1.0" ?>
<launch>
  <node name="service_node" pkg="mastering_ros_demo_pkg" type="demo_service_server" output="screen"/>
  <!-- <node name="client_node" pkg="mastering_ros_demo_pkg" type="demo_service_client" output="screen"/> -->
</launch>